eoscostarica/eosio-dashboard
# fix(webapp): telos testnet endpoint temp fix

### Telos Tesnet endpoint changed

### What does this PR do?

- Resolve #923 

### Steps to test

1. go to localhost and see testnet showing info

![image](https://user-images.githubusercontent.com/55892352/192596736-4705b413-adf4-4f07-8ee4-ef2e09469a05.png)

[Now this shows a reply](http://[testnet.telos.caleos.io/v1/chain/get_info](http://testnet.telos.caleos.io/v1/chain/get_info))
#### CheckList
